jujugui/static/gui/src/app/app.js

```diff
@@ -2110,15 +2108,30 @@ YUI.add('juju-gui', function(Y) {
       situations where auth is set outside the GUI (i.e., embedded).
 
       @method _getAuth
-      @param {String} service The service to retrieve auth info from.
      */
-    _getAuth: function(service) {
+    _getAuth: function() {
       var externalAuth = this.get('auth');
       if (externalAuth) {
         return externalAuth;
       }
       var users = this.get('users');
-      return users && users[service];
+      var user;
+      if (users) {
```

Comment:
Any time you're connected to the GUI but haven't yet logged into the charmstore. Which is most of the time for local dev.